### PR TITLE
Feature/societree

### PR DESCRIPTION
**Pull request by VamilP**
Society Tree Visualizer: The tool should represent a caste family tree with 72 nodes, where each node represents a primary family branch. Upon clicking on a node, it should expand to show the surnames associated with that branch, along with their respective deity and gotra information.

This JSON file contains information about different branches of Indian families, their Gotras (ancestral lineage), associated deities, and common surnames. Each entry includes:

Branch Name: The name of the family branch.
Gotra: The ancestral lineage associated with the branch.
Deity: The deity that the branch worships.
Surnames: A list of common surnames associated with the branch.

### CODE REVIEW

File: src/lang1.json
Comment (recursivezero) [inline src/lang1.json:1]: add one node which have more than 20 surnames and one with 0 nodes  and display it.

File: src/script.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    const cardContainer = document.getElementById("card-container");
+    cardContainer.innerHTML = "";
+
+    data.forEach((branch) => {
+      const card = document.createElement("div");
+      card.className = "card";
+
+      const surnamesHTML = branch.Surnames.map(
+        (surname) => `
+        <div class="surname-node">
```
Comment (recursivezero) [inline src/script.js]: use **details** and **summary** tag which have automatic close open feature. no need to done using js

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,167 @@
+@font-face {
```
Comment (recursivezero) [inline src/style.css]: create assets/fonts folder and put font there and use that location

File: src/README.md
Comment (recursivezero) [inline src/README.md:1]: Add how to start the project?

**Review by recursivezero — CHANGES_REQUESTED** [review]

**Comment by xkeshav** [comment]
resolve this conflicts and is that new PR?

**Comment by recursivezero** [comment]
resolve the conflicts

**Pull request merged**